### PR TITLE
CRN-612-show-snippet-of-notes-on-past-events

### PR DESCRIPTION
**Pull request by kmaid**

### CODE REVIEW

File: packages/react-components/src/molecules/Collapsible.tsx [hunk 1]
```diff
@@ -27,7 +47,7 @@ const Collapsible: React.FC<CollapsibleProps> = ({
           >
             {expanded ? chevronCircleUpIcon : chevronCircleDownIcon}
           </span>
-          {expanded ? 'Hide' : 'Show'}
+          {expanded ? 'Hide' : 'Show'} more
```
Comment (niezgoda) [inline packages/react-components/src/molecules/Collapsible.tsx:50]: image

Comment (niezgoda) [reply packages/react-components/src/molecules/Collapsible.tsx:50]: image

Comment (niezgoda) [reply packages/react-components/src/molecules/Collapsible.tsx:50]: I guess that it will be hard to make it "smarter"

**Review by niezgoda — APPROVED** [review]

File: packages/react-components/src/molecules/Collapsible.tsx (hunk 1 above)
Comment (kmaid) [reply packages/react-components/src/molecules/Collapsible.tsx:50]: Yeah, I don't really like it which is why i got beatriz to acknowledge and accept the problems in refinement. I think it be better to show the text in full or ask for an additional summary field.

**Pull request merged**